docs/curation-guidelines.md

```diff
@@ -90,3 +108,33 @@ We always prefer a reference to a version control system, but if you’re unable
 
 # Facets
 [TODO]
+
+# Appendix A - Package ecosystems
+
+## Maven
+|  | Canonical Location |  
+| --- | --- |
+| **License file**| Look in the -sources.jar.  The “Source” field on the definition will link to the -sources.jar. |
+| **Package metadata**|  Look on the .pom file (on central.maven.org/) for `</licenses>`.  Do not rely on https://mvnrepository.com/.  To navigate to central.maven.org/, click on the package version number on the definition.|
+| **Source repository**| Look on the .pom file (on central.maven.org/) for `</url>`.|
+
+## NPM
+|  | Canonical Location |  
+| --- | --- |
+|**License file**|  Look in the ClearlyDefined definition for the package files.| 
+|**Package metadata**|  Look on http://registry.npmjs.com/.  Do not rely on the license information on npmjs.com, because it is not version specific.  To find the registry, use this format: `http://registry.npmjs.com/[package name]/[version number]`.  Alternatively, you can look in the ClearyDefined definition for the package.json file and look for "license."|
```

Comment:
`Do not rely on the license information on npmjs.com...`
--> IMHO we should be more explicit and give an example. Since registry.npmjs.com is also on npmjs.com we want to make sure we we do not bring in any confusion.

Reply:
@pombredanne brings up a good point. I also think we should be more clear here.

Perhaps we shift focus to the component's version-specific _package.json_ if it's obtainable from _registry.npmjs.org_, similar to additional context we provided above for Maven POMs. 

I've seen when examining an NPM component on ClearlyDefined.io, that the blue "Download component" button points to a zip archive on _registry.npmjs.org_ (I.e. https://registry.npmjs.com/apache-bridge/-/apache-bridge-0.1.6.tgz). I don't know how else you can get the direct package easier without making an API call or using npm CLI.
 
From what I understand, it's the npmjs.org "License" meta field that we want to avoid because it is not version specific.

![image](https://user-images.githubusercontent.com/31491646/73205402-c71db700-40f5-11ea-848f-8aceb60387c6.png)

I want to say this is also a concern for others like NuGet and PyPi as well. It certainly is for GitHub too, as IIRC, the license recommendation seen at the top only conveys that of the latest version.

![image](https://user-images.githubusercontent.com/31491646/73205918-c33e6480-40f6-11ea-9d37-ded89d539b9d.png)

Reply:
Yes Brian, the license displayed on npmjs.org is not version specific - see https://npm.community/t/displaying-the-actual-license-of-the-node-module-version-on-npmjs-com/7954